web_responsive/static/src/less/main.less

```diff
@@ -4,6 +4,7 @@
 body {
     width: 100%;
     height: 100%;
+    overflow: hidden!important; // Important is to override base template
```

Comment:
Just removed overflow: hidden!important from main.less. Library drawer.3.2.0.js adds "overflow: auto" on close, also triggered on resize like @leorochael told us. This causes body to have it's own scrollbar and at small height screens we got 2 scrollbars at the same time. Committing that change on lib separately now...